spring-cloud-kubernetes-client-config/src/main/java/org/springframework/cloud/kubernetes/client/config/KubernetesClientConfigMapsCache.java

```diff
@@ -50,6 +50,10 @@ public void discardAll() {
 		CACHE.clear();
 	}
 
+	public static void discardCache() {
```

Comment:
It just seemed like extra work to instantiate a class to clear a static variable where if we just created a static method to do it things would be a bit cleaner

Reply:
Also please add all your comments and and I will clean everything up in this PR.  I will try to add some detailed explanation of certain changes once I am happy with the PR.

Reply:
Sure I can change it, its a fair point.  

Its just that the class already needs to be made public whether this method is added or not and then we already have a public method to clear the cache, so is it really exposing us any further? 

Since it just used in tests its not a big deal.

Believe me, you have every right to comment on PRs, you have done a lot of work!  And we appreciate it!